### PR TITLE
DOCS-3491 RUM FAQs to Guides Migration

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Migrates most RUM FAQs into guides, implements title case in guide titles, addresses some copy nits in guides.

### Motivation

✨ Q2 2022 Docs Hackathon ✨

### Additional Notes

Depends on https://github.com/DataDog/browser-sdk/pull/1597 being merged first so the link checker doesn't break.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/faq/content_security_policy.md [hunk 1]
```diff
@@ -17,17 +19,17 @@ Depending on the `site` option used to initialize [Real User Monitoring][2] or [
...
+If you are using Session Replay, make sure to allow workers with `blob:` URI schemes by adding the following `worker-src` entry:
 
 ```txt
 worker-src blob:;
 ```
 
 ## CDN bundle URL
 
-If you are using the CDN async or CDN sync setup for [Real User Monitoring][4] or [browser log collection][5], you should also add the following `script-src` entry:
+If you are using the CDN async or CDN sync setup for [Real User Monitoring][4] or [Browser Log Collection][5], you also add the following `script-src` entry:
```
Comment (alai97) [inline content/en/real_user_monitoring/faq/content_security_policy.md:32]:
```suggestion
If you are using the CDN async or CDN sync setup for [Real User Monitoring][4] or [Browser Log Collection][5], also add the following `script-src` entry:
```

**Comment by alai97** [comment]
@BenoitZugmeyer Yes, we're adding the deleted URLs as aliases to the nearest guide landing page according to the [Moving and Migrating Docs Pages](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/2174550131/Moving+Migrating+Docs+Pages) instructions!

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
 ---
-title: Proxy Real User Monitoring Data
-kind: faq
+title: Proxy Your Browser RUM Data
+kind: guide
```
Comment (kayayarai) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:3]:
```suggestion
kind: guide
aliases:
  - /real_user_monitoring/faq/proxy_rum_data/
```

File: content/en/real_user_monitoring/guide/session-replay-service-worker.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Allow Third-Party Service Worker for Session Replay
-kind: faq
+title: Allow Third-Party Service Workers For Session Replay
+kind: guide
```
Comment (kayayarai) [inline content/en/real_user_monitoring/guide/session-replay-service-worker.md:3]:
```suggestion
kind: guide
aliases:
  - /real_user_monitoring/faq/session_replay_service_worker/
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Added the aliases on the two renamed files.

**Review by kayayarai — APPROVED** [review]

**Pull request merged**